actions/delete_node_handler.go

```diff
@@ -156,10 +159,40 @@ func (h *deleteNodeHandler) Handle(ctx context.Context, action *castai.ClusterAc
 			if len(pods.Items) > 0 {
 				return true, fmt.Errorf("waiting for %d pods to be terminated on node %v", len(pods.Items), req.NodeName)
 			}
+			volumeAttachments, err := h.clientset.StorageV1().VolumeAttachments().List(ctx, metav1.ListOptions{})
+			if err != nil {
+				return true, err
+			}
+			if err != nil {
+				return true, fmt.Errorf("unable to list volume attachments for node %q err: %w", req.NodeName, err)
+			}
```

Comment:
Redundant block, only the first one will ever trigger (both err != nil)

Reply:
good catch, thanks